apps/aecore/src/aec_tx_pool.erl

```diff
@@ -407,7 +407,7 @@ check_candidate(Db, #dbs{gc_db = GCDb} = Dbs,
                 AccountsTree, Height, Gas, Acc) ->
     Tx1 = aetx_sign:tx(Tx),
     TxTTL = aetx:ttl(Tx1),
-    TxGas = aetx:gas(Tx1),
+    TxGas = aetx:gas(Tx1, Height),
```

Comment:
I see this can fail with badarith exception.
I see this is run in the worker process spawned `aec_block_generator:create_block_candidate`.
What prevents the worker process to terminate abnormally for oracle transaction with absolute TTL of state tree object not high enough for current chain?

Reply:
I see this fixed in 4f65906b (maybe before). Ok.